fiftyone/core/tags.py

```
@@ -578,26 +578,48 @@ def clear(self):
         """
         return self.delete(delete_all=True)
 
-    def count(self, filter: TemporalTagFilter | None = None) -> dict[str, int]:
-        """Counts temporal tag occurrences in this collection.
+    def count(
+        self,
+        filter: TemporalTagFilter | None = None,
```

Ruff (0.15.20)

[error] Function argument `filter` is shadowing a Python builtin

(A002)